config/web.js

```diff
@@ -130,7 +130,6 @@ const config = {
             '/',
             '/login',
             '/oauth/authorize',
-            '/register',
```

Comment:
Note: since the register route is removed here, the "No new registrations are accepted." doesn't appear and a 404 is shown instead:

<img width="1215" height="193" alt="Image" src="https://github.com/user-attachments/assets/6d66e242-79fc-4a2e-9aa2-ea1629aeac82" />